crates/provider/src/handlers/function_list.rs

```diff
@@ -51,7 +54,7 @@ pub async fn function_list_handler(req: HttpRequest) -> impl Responder {
     };
     log::info!("container_list: {:?}", container_list);
 
-    match get_function_list(container_list, namespace).await {
+    match get_function_list(container_list, namespace, &containerd_manager).await {
         Ok(functions) => HttpResponse::Ok().body(serde_json::to_string(&functions).unwrap()),
         Err(e) => HttpResponse::BadRequest().body(format!("Failed to get function list: {}", e)),
     }
```

Comment:
帮忙把所有丑不拉几的 match 弄成 map
返回类型改成 `actix_web::Result<HttpResponse>`
```suggestion
    get_function_list(container_list, namespace, &containerd_manager).await
        .map_err(|e|ErrorBadRequest(format!("Failed to get function list: {}", e)))
        .map(|functions| HttpResponse::Ok().json(functions))
```